qml/cover/CoverPage.qml

```diff
@@ -1,22 +1,52 @@
 import QtQuick 2.0
 import Sailfish.Silica 1.0
 
+import "../js/yleApi.js" as YleApi
+
 CoverBackground {
+    id: coverPage
     Label {
         id: label
-        anchors.centerIn: parent
-        text: qsTr("My Cover")
+        color: Theme.primaryColor
+        font.bold: true
+        text: qsTr("Näyttämö")
+        anchors {
+            horizontalCenter: parent.horizontalCenter
+            top: parent.top
+            topMargin: Theme.paddingMedium
+        }
     }
 
-    CoverActionList {
-        id: coverAction
-
-        CoverAction {
-            iconSource: "image://theme/icon-cover-next"
+    Column {
+        width: parent.width - 2 * Theme.paddingSmall
+        anchors.centerIn: parent
+        TextArea {
```

Comment:
Is there a reason for this to be a `TextArea` instead of `TextField` ? Can it contain multiple lines sometimes?